apps/site/components/Downloads/ReleaseOverview/index.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  CalendarIcon,
+  ClockIcon,
+  CodeBracketSquareIcon,
+  Square3Stack3DIcon,
+} from '@heroicons/react/24/outline';
+import { useTranslations } from 'next-intl';
+import type { FC, ReactNode } from 'react';
+
+import FormattedTime from '@/components/Common/FormattedTime';
+import type { NodeRelease } from '@/types';
+
+import styles from './index.module.css';
+
+type ItemProps = {
+  icon: React.JSX.Element;
```

Comment:
this type look strange we din't have something better ?

Reply:
We can use it directly from the `JSX` namespace;

```suggestion
  icon: JSX.Element;
```

Reply:
Not anymore: [react-19-upgrade-guide](https://react.dev/blog/2024/04/25/react-19-upgrade-guide#the-jsx-namespace-in-typescript)

Reply:
But I changed it to `FC<SVGProps<SVGSVGElement>>`, what do you think?

Reply:
I think the current approach seems better to use `React.JSX.Element`  🙇  I'll take a look at the migration documentation, thanks!

Reply:
https://github.com/nodejs/nodejs.org/blob/main/COLLABORATOR_GUIDE.md#best-practices-for-component-development-in-general "Avoid importing React, only import the modules from React that you need"

that's why it seems to me an odd type of typing. In absolute terms, it's good, but not according to our best practices.